include/deal.II/base/timer_tree.h

```diff
@@ -0,0 +1,423 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 1998 - 2020 by the deal.II authors
```

Comment:
This should only be for 2021.

Reply:
changed here https://github.com/dealii/dealii/pull/11939/commits/647d0da4b0f431358b92b969ce5b3fcf983abbda